codalab/worker/bundle_manager.py

```diff
@@ -311,6 +311,12 @@ def _schedule_run_bundles_on_workers(self, workers, user_owned):
             if user_owned:
                 workers_list = workers.user_owned_workers(bundle.owner_id)
             else:
+                if not self._model.get_user_parallel_job_quota_left(bundle.owner_id):
+                    logger.info(
+                        "User %s has no parallel job quota left, skipping job for now",
```

Comment:
It would be good if the function returned the actual quota and number of running jobs so we can report both numbers and the difference to give more information.

Reply:
The only case where this log  would print is if their number of runs is equal to the quota, so I don't see the value of printing both numbers  here, unless we want to print it for every run being started.

Reply:
Can you print out the one number then...User %s already has {} runs running...'
s/job/run